plugins/modules/docker_compose_v2.py

```diff
@@ -0,0 +1,923 @@
+      - When false, equivalent to C(docker-compose up --no-deps).
+      - When I(state) is C(pull) specify whether or not to also pull for linked services.
+      - When true, equivalent to C(docker-compose pull --include-deps).
+    type: bool
+    default: true
+  definition:
+    description:
+      - Compose file describing one or more services, networks and volumes.
+      - Mutually exclusive with I(files).
+    type: dict
+  recreate:
+    description:
+      - By default containers will be recreated when their configuration differs from the service definition.
+      - Setting to C(never) ignores configuration differences and leaves existing containers unchanged.
+      - Setting to C(always) forces recreation of all existing containers.
+      - When set to C(never), equivalent to C(docker-compose up --no-recreate).
+      - When set to C(always), equivalent to C(docker-compose up --force-recreate).
+    type: str
+    default: smart
+    choices:
+      - always
+      - never
+      - smart
+  build:
+    description:
+      - Use with I(state) C(present) to always build images prior to starting the application.
+      - Equivalent to C(docker-compose up --build).
+      - Images will only be rebuilt if Docker detects a change in the Dockerfile or build directory contents.
+      - If an existing image is replaced, services using the image will be recreated unless I(recreate) is C(never).
+    type: bool
+    default: false
+  pull:
+    description:
+      - Use with I(state) C(present) to always pull images prior to starting the application.
+      - Equivalent to C(docker-compose up --pull always).
+      - When a new image is pulled, services using the image will be recreated unless I(recreate) is C(never).
+    type: bool
+    default: false
+  nocache:
+    description:
+      - Use with the I(build) option to ignore the cache during the image build process.
+      - Equivalent to C(docker-compose build --no-cache).
+    type: bool
+    default: false
+  remove_images:
+    description:
+      - Use with I(state) C(absent) to remove all images or only local images.
+      - Equivalent to C(docker-compose down --rmi all|local).
+    type: str
+    choices:
+      - all
+      - local
+  remove_volumes:
+    description:
+      - Use with I(state) C(absent) to remove data volumes.
+      - Equivalent to C(docker-compose down --volumes).
+    type: bool
+    default: false
+  remove_orphans:
+    description:
+      - Remove containers for services not defined in the Compose file.
+      - Equivalent to C(docker-compose up --remove-orphans) or C(docker-compose down --remove-orphans).
+    type: bool
+    default: false
+  timeout:
+    description:
+      - Timeout in seconds for container shutdown when attached or when containers are already running.
+      - By default C(compose) will use a C(10s) timeout unless C(default_grace_period) is defined for a
```

Comment:
Should `compose` be `docker-compose` here?
```suggestion
      - By default C(docker-compose) will use a C(10s) timeout unless C(default_grace_period) is defined for a
```

Reply:
Indeed, it should be docker-compose.